docs/examples/jaeger-docker.md

```diff
@@ -7,8 +7,13 @@ otel4s library.
 
 ### Project setup
 
-Configure the project in the `build.sbt`:
+Configure the project using your favorite tool:
 
+@:select(build-tool)
+
+@:choice(sbt)
+
+Configure your `build.sbt`:
```

Comment:
Should I use `Add settings to the` rather than `Configure your` to preserve consistency between two examples?

Reply:
Not a big deal, but I like "add settings to"